Preserve type Annotations (#1730)
strawberry-graphql/strawberry: RELEASE.md

Release type: minor

> Add StrawberryAnnotated, a container type to preserve `typing.Annotated` data when converting types to StrawberryTypes

Reviewer: ```suggestion
This release adds support for `typing.Annotated` so that is preserved  when converting types to StrawberryTypes
```
I would rephrase a little to emphasize the functionality we are adding rather than what changed internally :)

Author: Sounds good, thank you
